src/sockets/commands/user/mute.ts

```diff
@@ -1,7 +1,7 @@
 import { Command } from '../types';
 import { SocketUser } from '../../types';
 import { sendReplyToCommand } from '../../sockets';
-import userAdapter from '../../../databaseAdapters/user';
+import mongoDbAdapter from '../../../databaseAdapters/mongoose';
```

Comment:
We shouldn't be referring to mongoose from these files. It should call some GetUserAdapter() and it just happens to return mongoose. This makes it easy to change in the future if we need to.